### PR TITLE
Add type to PR template

### PR DESCRIPTION
**Pull request by l2dy**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 
 <!-- (delete all below for minor changes) -->
 
+###### Type(s)
+<!-- update (title contains ": U(u)pdate to") and submission (new Portfile) are auto-detected, replace [ ] with [x] to select -->
+- [ ] bugfix
+- [ ] enhancement
+- [ ] security fix
```
Comment (neverpanic) [inline .github/PULL_REQUEST_TEMPLATE.md]: We can additionally offer auto-detection if the commit message matches `CVE-\d{4}-\d+`.

**Comment by l2dy** [comment]
I'll merge this when the PR bot is ready.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,8 +1,15 @@
 ###### Description
 
 
+<!-- Use "skip notification" (surrounded with []) to avoid notifying maintainers -->
```
Comment (mojca) [inline .github/PULL_REQUEST_TEMPLATE.md]: I have some problems understand what exactly I'm supposed to do here to make this work. Just use
```
[skip notification]
```
on a line on its own?

Should this be uncommented or may I just modify the existing sentence inside the comment?

Comment (l2dy) [inline .github/PULL_REQUEST_TEMPLATE.md]: As long as the PR body contains `[skip notification]` it would work (even in a comment).

Comment (l2dy) [inline .github/PULL_REQUEST_TEMPLATE.md]: Any suggestion on wording?

Comment (neverpanic) [inline .github/PULL_REQUEST_TEMPLATE.md]: Wording works for me, no strong opinion on it.

**Comment by mojca** [comment]
I have no good idea about the wording either and this is pure nitpicking anyway, just ignore.

I started writing a few proposals, but all of them sound equally bad :)

**Pull request merged**